shared/js/background/storage/https.es6.js

```diff
@@ -22,6 +25,9 @@ class HTTPSStorage {
             const etag = settings.getSetting(`${listCopy.name}-etag`) || ''
 
             return this.getDataXHR(listCopy.url, etag).then(response => {
+                // Set the lastUpdate time.
+                settings.updateSetting(`${listCopy.name}-lastUpdate`, Date.now())
```

Comment:
Can we trust the system clock here? If the user has their clock set in the future, then resets it to the correct time, updates will be blocked until we reach the previous timestamp.

An alternative would be to use the server-sent timestamp in the `Date` header of the response.

Reply:
I suppose that's true. If the user set their system clock far in the future, installed the extension, set their system clock back again, and then took care to never run the extension for more than the update period (30 mins for TDS, 12 hours for HTTPS) the extension configurations would never be updated.

That doesn't seem like an especially likely scenario though. I suppose it doesn't harm to attempt to parse the Date header, if available and then choose the lower out of Date.now() and the response header date. Will update that now.